Makefile

```diff
@@ -66,11 +66,11 @@ doclinter:
 
 .PHONY: test
 test:
-	@$(PYTHON) -X dev -m pytest -v $(TEST)
+	@$(PYTHON) -X dev -X warn_default_encoding -m pytest -v $(TEST)
```

Comment:
Is it okay to pass this option to the old pythons?

Reply:
Yes, see https://docs.python.org/3/using/cmdline.html#cmdoption-X

> It also allows passing arbitrary values and retrieving them through the [sys._xoptions](https://docs.python.org/3/library/sys.html#sys._xoptions) dictionary.

So for Python 3.9 and earlier the value is just ignored.

A

Reply:
Thank you for your explanation. Okay, let's merge :-)

Reply:
Thanks!